From 73cd928fe1a9c2abbe6f28844a7fc695ad11d7cf Mon Sep 17 00:00:00 2001
From: Peter Lama <peterldev94@gmail.com>
Date: Fri, 24 Aug 2018 17:25:47 -0700
Subject: [PATCH] Fix issue #39: PyUnicode_AsUTF8 returns const char*

Even with python 2, it doesn't seem like char* is needed

--- Inventor/nodes/SoMarkerSet.i
+++ Inventor/nodes/SoMarkerSet.i
@@ -5,7 +5,7 @@
         short WIDTH, HEIGHT;
         size.getValue(WIDTH, HEIGHT);
         short BYTEWIDTH = (WIDTH + 7) / 2;
-        char* coin_marker;
+        const char* coin_marker;
 #ifdef PY_2
         if (PyString_Check(string))
         {
@@ -49,4 +49,4 @@
         delete[] bitmapbytes;
         bitmapbytes = NULL;
     }
-}
\ No newline at end of file
+}
--- Inventor/sensors/SoSensor.i
+++ Inventor/sensors/SoSensor.i
@@ -4,7 +4,7 @@ SoSensorPythonCB(void * data, SoSensor * sensor)
 {
   PyGILState_STATE gil = PyGILState_Ensure();
   swig_type_info * swig_type = 0;
-  char * sensor_cast_name = NULL;
+  const char * sensor_cast_name = NULL;
   PyObject * func, * arglist;
   PyObject * result, * pysensor;
 
